includes/replace.form.inc

```diff
@@ -31,6 +31,8 @@ function islandora_find_replace_replace_form($form, &$form_state, $find_replace)
 
   foreach (unserialize($find_replace['find_results']) as $option) {
     $options[$option['object']] = array(
+      'pid' => l($option['object'], 'islandora/object/' . $option['object'],
```

Comment:
Hi! Nice. Do we need both to have links to the same?

Reply:
What i mean is maybe we just need a single link. Does not change anything other than user experience.

Reply:
Hard to say. I personally look for links under the PID, but others might go for the title instead. Any reason it's better not to have it in both places?

Reply:
I would say there is no real huge reason but i don't it confuses people to have two links to the same. Up to you really. As long as we have a link i'm ok with it!

Reply:
I'm happier to leave it as two links if you don't mind! :)